sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -325,7 +325,7 @@
       "post": {
         "summary": "Query an IoT hub to retrieve information regarding device twins using a SQL-like language.",
         "description": "Query an IoT hub to retrieve information regarding device twins using a SQL-like language. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-query-language for more information. Pagination is supported. This returns information about device twins only.",
-        "operationId": "RegistryManager_QueryIotHub",
+        "operationId": "Query_Twins",
```

Comment:
I wonder if we should call this Query_GetTwins. With Query_Twins, the method will show up in the generated file as simply Twins(). Not a blocker, but I thought I would open for discussion.

Reply:
Good point! I missed that when I suggested this name

Reply:
agreed, renamed